src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -127,7 +134,7 @@ class BuiltinLower : public StmtExprMutator {
         auto storage_scope = Downcast<PointerType>(op->buffer_var->type_annotation)->storage_scope;
         if (dev_type->value == kDLCPU && storage_scope == "global") {
           int32_t constant_size = op->constant_allocation_size();
-          if (constant_size > 0 && constant_size * nbytes < runtime::kMaxStackAlloca) {
+          if (constant_size > 0 && constant_size * nbytes < kMaxStackAlloca) {
```

Comment:
Maybe you could also change the `<` to `<=`.  Intuitively, an allocation of size `kMaxStackAlloca` should still be allowed on the stack, but the `<` would prevent it.

Reply:
I changed the code accordingly